sample/src/main/res/values/dimens.xml

```diff
@@ -19,4 +19,5 @@
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
     <dimen name="fab_margin">16dp</dimen>
+    <dimen name="sd_label_max_width">200dp</dimen>
```

Comment:
**Reporter**: Android Lint
**Rule**: Performance
**Severity**: WARN
**File**: sample/src/main/res/values/dimens.xml L22

UnusedResources: Unused resources
The resource `R.dimen.sd_label_max_width` appears to be unused
Unused resources make applications larger and slow down builds.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-1781178467>*